test_read_behavior.py

```
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+"""
+Test script to demonstrate read_process_output behavior on closed processes
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)